README.md

```diff
@@ -164,6 +166,9 @@ Freeside welcome contributions to these lists from staff and students at the Uni
 24. [University of Hull - Hull's digital repository, Hydra](https://hydra.hull.ac.uk/)
 25. [Visualisation and Audibilization of Sorting Algorithms](https://panthema.net/2013/sound-of-sorting/) - Visualize different sorting algorithms and listen the sound of sorting.
 26. [Sentdex Youtube Channel](https://www.youtube.com/c/sentdex) - Youtube channel to learn Machine learning, Deep Learning, Natural Language Processing and python.
+27. [Forage](https://www.theforage.com/) - Offers 5-6 hours virtual work experience programs.
```

Comment:
Looks useful, but not sure this belongs in this section?

Reply:
If a new section is to be added, it'd be worth having multiple resources for it ready, for now I'd say it is fine where it is or in business/career